packages/node-sdk/paima-utils-backend/src/cde-access-internals.ts

```diff
@@ -36,26 +36,26 @@ import type {
 
 /* Functions to retrieve CDE ID: */
 
-export async function getCdeIdByName(
+export async function getcdeNameByName(
```

Comment:
I don't think this function does anything anymore because it takes the name as an input and returns the name as the output